### PR TITLE
Adds departmental protolathe items, techweb tweaks

### PR DESCRIPTION
**Pull request by 81Denton**
:cl: Denton
add: Added new available items to all departmental protolathes. Keep an eye out for the new "Basic Tools" and "Basic Security Equipment" research nodes.
tweak: Made the AI surveillance upgrade available through the "Illegal Technology" research node.
tweak: Created a "Tool Designs" lathe subsection and moved most tools into it.
balance: Made defibrillators easier to print by moving them to the basic biotechnology research node.
fix: Health analyzers are now properly printable once researched.
/:cl:

tl;dr I've heard a fair amount of complaints about departmental lathes that can be summed up with "cluttered sections, too few useful or interesting items in some lathes". 
I'm trying to improve it with the following:

A) Added a new category called "Tool Designs" and grouped all tools into it. That way, they no longer clutter up the Equipment/Medical Designs sections.

B) New lathe contents:

**Medical**: 
- Surgical tools, included in the "Basic Tools" node. 
These fit way better into the Medical/RnD lathes instead of a public one that also prints dishware.
- Basic 50u and large 100u beakers, roundstart.
- Medical spray bottles, printable with basic biotech.
- Compact defibrillators, printable with Advanced Biotech. 
They're fairly expensive, but obviously superior to regular defibs.

**Service**:
- Botanical tools, printable with Basic Tools.
- Spray bottles and bear traps, printable with sanitation tech.
- Basic 50u and large 100u beakers, roundstart.

**Security**:
- Seclites, pepper sprays (empty), energy bolas, zip ties, evidence bags, flashes. Printable with the new "Basic Security Equipment" node. 
I've had a look at weapons as well, but the ones that aren't printable/ingame yet are either ballistics or unbalanced.

**Science**:
- Mechanical/electric tools, printable with Basic Tools.
- Illegal technology unlocks the AI surveillance software upgrade. 
Installing it on an AI lets it listen to conversations through cameras.

**Engineering**:
- Artificial bluespace crystals, alien alloy. 
This is neccessary in case engineers have to rebuild a teleporter, but miners haven't brought back any bluespace crystals.
- Mechanical/electric and atmos tools, printable with Basic Tools.

**All lathes**:
- Hand labelers, destination taggers and wrapping paper. 
This lets crew mail packages when the mail office isn't manned (which is 24/7).
- Illegal technology unlocks Donksoft refill packs. 
You can't build the Donksoft vendor without refill packs; not including them was an oversight in my earlier PR.

### CODE REVIEW

**Comment by fen0s** [comment]
DONKSOFT GETTING SOME LOVE FINALLY
MERGE

**Comment by 81Denton** [comment]
All defibs lose a bunch of charge and let you shock people once they're emagged. Takes both hands though.

**Comment by Tlaltecuhtli** [comment]
donk soft can be ordered by cargo and i dont think making it so you can every round make 10-20 donk soft a good idea

**Comment by fen0s** [comment]
It's like sci researches illegal every round, mhmmm

**Comment by 81Denton** [comment]
Let's see, donksoft refill packs have a chance to pop up in the toy and contraband crate. That might be good enough to not include them with illegal tech?

File: code/modules/research/designs/autolathe_designs.dm [hunk 1]
```diff
@@ -5,18 +5,20 @@
 /datum/design/bucket
 	name = "Bucket"
 	id = "bucket"
-	build_type = AUTOLATHE
+	build_type = AUTOLATHE | PROTOLATHE
 	materials = list(MAT_METAL = 200)
 	build_path = /obj/item/reagent_containers/glass/bucket
-	category = list("initial","Tools")
+	category = list("initial","Tools","Equipment")
```
Comment (octareenroon91) [inline code/modules/research/designs/autolathe_designs.dm]: Should this be `category = list("initial","Tools","Tool Designs")` ?

**Comment by octareenroon91** [comment]
If a full donksoft load is too much to make so freely available, maybe make a `/obj/item/vending_refill/donksoft/safety` without the automatics?

**Comment by Tlaltecuhtli** [comment]
the donk soft its fine as it is (orderable from cargo)

File: code/modules/research/designs/autolathe_designs.dm (hunk 1 above)
Comment (81Denton) [reply code/modules/research/designs/autolathe_designs.dm]: Yeah, that would make sense.

**Comment by Mickyan** [comment]
medical could use empty medical sprays

**Comment by octareenroon91** [comment]
>medical could use empty medical sprays

Basic spraybottles are also hard to find.

File: code/modules/research/designs/misc_designs.dm [hunk 2]
```diff
@@ -397,4 +326,68 @@
...
+	build_path = /obj/item/restraints/legcuffs/bola/energy
+	category = list("Equipment")
+	departmental_flags = DEPARTMENTAL_FLAG_SECURITY
+	
+/datum/design/zipties
+	name = "Zipties"
+	desc = "Plastic, disposable zipties that can be used to restrain temporarily but are destroyed after use."
+	id = "zipties"
+	build_type = PROTOLATHE
+	materials = list(MAT_PLASTIC = 250)
```
Comment (octareenroon91) [inline code/modules/research/designs/misc_designs.dm]: Could I bother you to add this to the zipties object type?

**Comment by fen0s** [comment]
(gosh, guys, are you No Fun Allowed police or what?)
The problem is that cargo is incompetent almost every round, and most likely they'll never do your request and just will order thousand gun crates. If you won't add refillers, then remove donksoft boards from Illegal Tech so it's not useless.

**Comment by Bombpants** [comment]
Did you remove surgical tools from auto lathes? Don’t remove the gray tides source of chainsaws.

**Comment by Bombpants** [comment]
Did you remove surgical tools from auto lathes? Don’t remove the gray tides source of chainsaws.

**Comment by 81Denton** [comment]
Nah, they are still available at autolathes too

File: code/modules/research/designs/misc_designs.dm (hunk 2 above)
Comment (81Denton) [reply code/modules/research/designs/misc_designs.dm]: How do you mean exactly?

**Comment by 81Denton** [comment]
Alright, I added medical sprays (medlathe) and spray bottles (service lathe).

**Comment by octareenroon91** [comment]
I have a feeling plastic flaps will be routinely greytided soon and I have no particular issue with that possibility.

**Comment by Vidiveni** [comment]
Bear traps to service lathe imo, unless they are there already.

**Comment by octareenroon91** [comment]
Bear traps are a bit strong. Maybe under the `"weaponry"` node it'd be OK.

**Comment by Vidiveni** [comment]
Ifeel service deserves bear traps since their lathe is so lackluster in content, and the other departments have their own weaponry already. But it should be in the techweb and not be a roundstart thing.

**Comment by 81Denton** [comment]
Bear traps will be available for 5000 metal/1000 titanium once advanced sanitation tech is researched

**Comment by Tlaltecuhtli** [comment]
can you make bear traps insta kill bears (or maybe damage simple mobs) so there is a reason other than grief for them to exist

**Comment by Vidiveni** [comment]
Don't know what cost would be just right, but metal+titanium sounds good. I like the idea of bear traps hurting simple mobs. They are also useful for slowing stun-immune people on drugs (hulks). 

Also, how would people feel about adding the **dragnet** to late techweb security techfabs? There is currently no legit way of getting more of those, that I know of. Or maybe from supply crates instead.

**Comment by optimumtact** [comment]
If there's no legit way of getting an item outside of admin spawn do not add it in this PR or it will delay the whole thing

**Pull request merged**